src/nlp/Preprocessing.py

```diff
@@ -253,3 +254,37 @@ def correct(self):
       corrected = doc._.outcome_spellCheck
 
     return corrected
+
+  def handleAbbreviations(self, abbrDatabase, type):
+    """
+      Performs automatic correction of abbreviations and returns corrected text
+      @ In, abbrDatabase, pandas dataframe, dataframe containing library of abbreviations 
```

Comment:
It may worth to describe the structure of abbrDatabase here.

Reply:
added